41_snapshots-btrfs

```diff
@@ -122,6 +122,25 @@ boot_uuid_subvolume=$(awk -F":" 'match($1, /(^[ \t]+UUID)/) {sub(/^[ \t]+/, "",
 boot_hs=$(${grub_probe} --device ${boot_device} --target="hints_string" 2>/dev/null) # hints string
 boot_fs=$(${grub_probe} --device ${boot_device} --target="fs" 2>/dev/null) # Type filesystem of boot device
 
+# Enable LUKS encrypted devices support
+case "$(echo "$GRUB_BTRFS_ENABLE_CRYPTODISK" | tr '[:upper:]' '[:lower:]')" in
+    true)
+            list_insmods=()
+            list_insmods+=("insmod gzio")
+            list_insmods+=("insmod part_gpt")
+            list_insmods+=("insmod cryptodisk")
+            list_insmods+=("insmod luks")
+            list_insmods+=("insmod gcry_rijndael")
+            list_insmods+=("insmod gcry_rijndael")
```

Comment:
I can't speak to the bigger picture of what should and shouldn't be added with `insmod`, but this particular line is duplicated. Should it be duplicated?

Reply:
Sorry for the late reply. Honestly I merely copy-pasted what I saw in my grub.cfg , and apparently also in the related issue the insmod list for the user that opened it [contains the same duplication](https://github.com/Antynea/grub-btrfs/issues/260#issuecomment-1373716084) . Anyway I tried to boot a snapshot without the duplication and it works, so I don't really know if it's best to do the same duplication as "default" grub does or not